packages/react-components/react-motion-components-preview/stories/src/Slide/SlideSnappy.stories.md

```diff
@@ -0,0 +1,13 @@
+The `SlideSnappy` variant has a faster duration.
```

Comment:
Description is not linked to the story i.e. `parameters` are missing

Reply:
Done.